<launch>

	<node pkg="uchile_safety" type="safety.py" name="safety" ns="/bender/nav/" respawn="true" respawn_delay="2" output="screen">

	 	<rosparam command="load" file="$(find uchile_safety)/config/safety.yaml"/>

	 	<!-- odometry -->
	 	<remap from="~odom" to="/bender/nav/odom"/>

	 	<!-- input sensors -->
	 	<remap from="~scan_front" to="/bender/sensors/laser_front/scan_filtered"/>
	 	<remap from="~scan_rear"  to="/bender/sensors/laser_rear/scan_filtered"/>
	 	<remap from="~scan_rgbd"  to="/bender/sensors/rgbd_head/depth/scan"/>

	 	<!-- cmd_vel -->
	 	<remap from="~input"  to="/bender/nav/mux/cmd_vel"/>
	 	<remap from="~output" to="~cmd_vel"/>

	 	<!-- This remmap exists for the follow skill only -->
	 	<remap from="~triggered" to="/bender/nav/low_level_mux/obstacle"/>
	</node>

</launch>